Week3/homework/2-step3.js

```diff
@@ -2,7 +2,47 @@
 
 function threeFive(startIndex, stopIndex, threeCallback, fiveCallback) {
   const values = [];
-  // Add your code here
+
+  let arr;
+  if (startIndex > stopIndex) {
+    arr = startIndex - stopIndex;
+    for (let i = 0; i <= arr; i++) {
+      values[i] = stopIndex;
+
+      stopIndex++;
+    }
+  } else if (startIndex < stopIndex) {
+    arr = stopIndex - startIndex;
+    for (let i = 0; i <= arr; i++) {
+      values[i] = startIndex;
+
+      startIndex++;
+    }
```

Comment:
I Really Liked this `if` statement :slightly_smiling_face:, you can improve it within a half lines of code like so:
```javascript
const arr = startIndex > stopIndex ? startIndex - stopIndex : stopIndex - startIndex; 
  for (let i = 0; i <= arr; i++) {
    values[i] = stopIndex; // Oops
    stopIndex++;
  }
```
But also in any case `arr` is not in use :upside_down_face:, so Now you may know how to solve the Issue?